package.json

```diff
@@ -10,5 +10,10 @@
   "packageManager": "pnpm@7.5.2",
   "volta": {
     "node": "17.9.1"
+  },
+  "pnpm": {
+    "patchedDependencies": {
+      "@woocommerce/components@10.0.0": "patches/@woocommerce__components@10.0.0.patch"
```

Comment:
I learned that we can create these patches using https://pnpm.io/cli/patch and https://pnpm.io/cli/patch-commit. Do you have a link, or could you briefly explain a workflow for updating patch files? e.g. when we update @woocommerce/components@10.0.0 te e.g. 11.0.0, what should we do with the patch file? I guess I'm not the only one who doesn't know, so maybe the explanation or a link to a blog post or so could be helpful in the P2 post you are preparing.

Reply:
I'm not sure there's a workflow for that. The package version needs to be exact, and I think that's because the patched code can change any time, even with a patch fix. Therefore, when updating, you'll probably need to patch again on the current, updated code. Maybe it's also kind of supposed that after an update you often no longer need the patch.

Do you think this is a problem? I'm trying to find what Yarn does, as they've been using the patch protocol for a long time already.

Reply:
I don't think it is a problem. The previous patch is a kind of guide for doing the new one so it seems straightforward. I just wanted to know what to do when I do the update.